### DISCUSSION TITLE
Commit Message as part of the label on History Report

### DISCUSSION THREAD

**Discussion by matt-franceschini-fw** [LironEr/bundlemon#109] [Ideas] [answered]
The history report feature would greatly benefit from having more information about the commit in the popover. It is very hard to pinpoint which commit lead to the increase/decrease in size. Can we pull information from the commit (ideally the message, as it is human readable)?

**Comment by LironEr** [comment, marked as answer]
Good idea, I'm planning to release a major version soon.

Hopefully, after the release, I will add this feature.

I opened an issue about it https://github.com/LironEr/bundlemon/issues/116

**Reply by LironEr** [reply]
Feature available in [bundlemon@2.0.0-rc.1](https://github.com/LironEr/bundlemon/releases/tag/bundlemon%402.0.0-rc.1)
